snap/hooks/post-refresh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+# Pre-refresh hook for charmed-mongodb snap
+
+set -eux
+
+export CONF="${SNAP_DATA}"/etc/mongod
```

Comment:
nit, for this and the following occurrences - e.g: `chown root:snap_daemon "${CONF}"` 
```suggestion
export CONF="${SNAP_DATA}/etc/mongod"
```